Content.Server/AutoVote/AutoVoteSystem.cs

```diff
@@ -0,0 +1,57 @@
+using Robust.Shared.Configuration;
+using Content.Server.Voting.Managers;
+using Content.Shared.GameTicking;
+using Content.Shared.Voting;
+using Content.Shared.CCVar;
+using Robust.Server.Player;
+using Content.Server.GameTicking;
+
+namespace Content.Server.AutoVote;
+
+public sealed class AutoVoteSystem : EntitySystem
+{
+    [Dependency] private readonly IConfigurationManager _cfg = default!;
+    [Dependency] public readonly IVoteManager _voteManager = default!;
+    [Dependency] public readonly IPlayerManager _playerManager = default!;
+
+    public bool _shouldVoteNextJoin = false;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<RoundRestartCleanupEvent>(OnReturnedToLobby);
+        SubscribeLocalEvent<PlayerJoinedLobbyEvent>(OnPlayerJoinedLobby);
+    }
+
+    public void OnReturnedToLobby(RoundRestartCleanupEvent ev)
+    {
+        CallAutovote();
+    }
+
+    public void OnPlayerJoinedLobby(PlayerJoinedLobbyEvent ev)
+    {
+        if (_shouldVoteNextJoin)
+        {
+            CallAutovote();
+            _shouldVoteNextJoin = false;
+        }
```

Comment:
```suggestion
        if (!_shouldVoteNextJoin)
            return;

        CallAutovote();
        _shouldVoteNextJoin = false;
```

Reply:
now we have to wait another 5 minutes for tests!